<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI Assigner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ql_paramsDirPath">
        <property name="text">
         <string>Path to parameter directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="qle_paramsDirPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="qpb_browseParams">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ql_cifFilePath">
        <property name="text">
         <string>Path to Cif file with the structure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="qle_cifFilePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="qpb_browseCif">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="qtbv_assignment"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="qpb_assign">
        <property name="text">
         <string>Assign</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="qa_actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTracing">
    <property name="title">
     <string>Tracing</string>
    </property>
    <addaction name="separator"/>
    <addaction name="qa_setupTracepoints"/>
    <addaction name="qa_viewLastTrace"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="qa_assignmentParameters"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTracing"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="qa_actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="qa_setupTracepoints">
   <property name="text">
    <string>Setup tracepoints</string>
   </property>
  </action>
  <action name="qa_viewLastTrace">
   <property name="text">
    <string>View last trace</string>
   </property>
  </action>
  <action name="qa_assignmentParameters">
   <property name="text">
    <string>Assignment parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
